src/CacheTower/Providers/FileSystem/FileCacheLayerBase.cs

```diff
@@ -136,6 +147,23 @@ public async Task SaveManifestAsync()
 				ManifestLock.Release();
 			}
 		}
+		public void SaveManifest()
```

Comment:
Missing the XML doc. While saying that though, not sure if this should be a public method as it is only needed for synchronous dispose.